extensions/css-language-features/server/src/pathCompletion.ts

```diff
@@ -144,27 +144,24 @@ const isDir = (p: string) => {
 };
 
 function pathToReplaceRange(valueBeforeCursor: string, fullValue: string, fullValueRange: Range) {
-	let replaceRange: Range;
```

Comment:
Please revert the unrelated style changes. You can submit them as a separate PR if you feel they are worthwhile but they should not be part of this PR

Reply:
@mjbvz there is a previous pr opened to fix the issue on localisation key to match with the correct property. It might be related with the bug in how the selection boundaries are validated (potentially affects settings code).

Is it relevant to change other files in the project just to follow the lowercase  pattern used? (whitespace instead of whiteSpace)? these files are creating constants and being used properly (it doesn't affect the bug).

should be a different pr (refactor)? unless the bug is related to that